### PR TITLE
Vareditting the dir var now calls setDir()

### PR DESCRIPTION
**Pull request by ShizCalev**
Fixes stuff getting out of sync in the case of things like chairs, infrared sensors, conveyor belts, ect.

Shouldn't have any issues, though for some reason I think there's probably a different proc up the atom movement chain that this should be calling instead.

### CODE REVIEW

**Review by SpaceManiac — DISMISSED** [review]
setDir seems right for this

**Comment by ninjanomnom** [comment]
I pretty commonly need to change the dir value directly and avoid the setDir call when debugging.

File: code/game/atoms.dm [hunk 1]
```diff
@@ -523,6 +523,12 @@
 		if("color")
 			add_atom_colour(color, ADMIN_COLOUR_PRIORITY)
 
+		if("dir")
+			var/do_setdir = alert(usr, "use setDir?", src, "Yes", "No")
+			if(do_setdir == "Yes")
+				setDir(var_value)
```
Comment (ninjanomnom) [inline code/game/atoms.dm]: Calling setDir after the value has already changed wont play nicely with functionality which expect the existing dir value to be the old one.

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

**Comment by ShizCalev** [comment]
rip

**Pull request closed without merging**